packages/terra-functional-testing/src/services/wdio-visual-regression-service/VisualRegressionLauncher.js

```diff
@@ -11,8 +11,24 @@ class VisualRegressionLauncher {
    * @param {Object} options.locale - The locale being tested.
    * @param {Object} options.theme - The theme being tested.
    */
-  constructor(options) {
-    this.compare = new LocalCompare(options);
+  constructor(options = {}, _capabilities, config = {}) {
+    const { launcherOptions } = config;
+    const { locale, theme } = launcherOptions || {};
+
+    /**
+     * Always use the launcher options from the test runner if available before using
+     * the options passed thru the service in wdio.conf.js. The reason is because the
+     * service options set using env is cached and are unreliable if these options
+     * are changed dynamically by the test runner.
+     *
+     * Reference: https://github.com/webdriverio/webdriverio/issues/6411
+     */
+    const visualRegressionOptions = {
+      ...(locale || options.locale) && { locale: locale || options.locale },
+      ...(theme || options.theme) && { theme: theme || options.theme },
```

Comment:
Should this also be passing the `updateScreenshots` config value?

Reply:
Currently, it is being passed down to the [global.browser.checkElement](https://github.com/cerner/terra-toolkit/blob/main/packages/terra-functional-testing/src/commands/validates/screenshot.js#L21) command. Yes, it would be more consistent for Visual Regression to get `updateScreenshots` directly from the CLI options.

https://github.com/cerner/terra-toolkit/pull/583/commits/2251ca53df2fa449f3ced6b1ef0d2d8f1b25e385

I will make a second update to pass it to the VR service options.